lib/plug/cowboy/translator.ex

```diff
@@ -20,40 +20,78 @@ defmodule Plug.Cowboy.Translator do
 
   ## Ranch/Cowboy
 
-  defp translate_ranch(
-         min_level,
-         _ref,
-         extra,
-         pid,
-         {reason, {mod, :call, [%Plug.Conn{} = conn, _opts]}}
-       ) do
-    if non_500_exception?(reason) do
-      :skip
-    else
+  if Version.match?(System.version(), "~> 1.6") do
```

Comment:
I think we can do this:

```elixir
if Version.match?(System.version(), "~> 1.6") do
  defp ok(message, metadata) do
    {:ok, message, metadata}
  end
else
  defp ok(message, _metadata) do
    {:ok, message}
  end
end
```

And then we do:

```elixir
ok(
  [
    inspect(pid),
    " running ",
    inspect(mod),
    extra,
    " terminated\n",
    conn_info(min_level, conn)
    | Exception.format(:exit, reason, [])
 ],
 [crash_reason: reason]
)
```

The advantage is then that we won't have to rewrite whole functions.

Reply:
things got weird with github being down for a bit, but I've amended the commit for excluding metadata